dockerfiles/google/Dockerfile

```diff
@@ -93,6 +93,15 @@ RUN curl -L -o ./tph.tar.gz \
         && mkdir -p /root/.terraform.d/plugins/ \
         && mv terraform-provider-helm_v${TERRAFORM_PROVIDER_HELM_VERSION} /root/.terraform.d/plugins/
 
+COPY terraform-plugins /terraform-plugins/
+
+RUN cd /terraform-plugins \
+    && terraform init \
+    && find /terraform-plugins \
```

Comment:
Is this needed? What is it for?

Reply:
I can't say for Alfredo, but this leaves a nice list with all installed TF plugins in CI logs:
https://gitlab.com/gpii-ops/exekube/-/jobs/99256670